### DISCUSSION TITLE
Enhancer For Youtube

### DISCUSSION THREAD

**Discussion by Aditya-Pathania** [Floorp-Projects/Floorp#1208] [Feature Request]
"Enhancer For Youtube" extention is not working on Floorp. It works on Firefox, but still no progress on Floorp.
Please, if you could see as to why, this discrepancy is occuring, and fix it, so that people can use that extention on Floorp as well.

**Comment by creeper-0910** [comment]
Does that mean I can't install it?

**Comment by CutterKnife** [comment]
ESR issues. An English version does not yet exist, but documentation is available:
https://docs.floorp.app/ja/issues/latest-extension/
You can configure Floorp to not check for compatibility with add-ons, but they cannot guarantee that it will work.
